src/ui/components/SessionContextAdapter.tsx

```diff
@@ -30,8 +30,7 @@ export default function SessionContextAdapter({
   const accessToken = useAppSelector(getAccessToken);
   const sessionId = useAppSelector(getSessionId);
   const duration = useAppSelector(getRecordingDuration)!;
-  const receivedPoints = useAppSelector(getPoints);
-  const endpoint = receivedPoints[receivedPoints.length - 1];
+  const endpoint = useAppSelector(getEndpoint);
```

Comment:
The only place we cared about these points was here, and we only cared about the endpoint. We can just store that in Redux instead.